tests/Feature/ApiMismatchRouteTest.php

```diff
@@ -62,21 +62,133 @@ public function test_single_mismatch_returns_correct_data()
             );
     }
 
-    public function test_multiple_mismatches_returns_correct_number_of_items()
+    public function test_multiple_mismatches_do_not_return_edited_or_expired()
     {
         $import = ImportMeta::factory()
         ->for(User::factory()->uploader())
         ->create();
 
-        $mismatches = Mismatch::factory(3)->for($import)->create();
+        $expiredImport = ImportMeta::factory()
+        ->for(User::factory()->uploader())
```

Comment:
Totally a nitpick, but I wonder if we should add a phpcs rule for indentation levels. Visually it bugs me, but as long as the code isn't broken, it's not a hill I'm prepared to die over.

Reply:
I couldn't find a phpcs rule that checks for the correct indentation level in this case. PSR-2 only defines that there must be at least 8 spaces, but it would also be happy with 13. I stopped searching the internet after 30 minutes.

Still fixed the indentation, thx.